app/views/layouts/_loginerror.html.erb

```diff
@@ -0,0 +1,7 @@
+<%- flash.each do |name, msg| -%>
+	<div class="alert alert-<%= name == :notice ? "success" : "danger" %>">
+	 <a href="" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+	<%= content_tag :div, msg, :id => "flash_#{name}" if msg.is_a?(String) %>
+	</div>
+	<%end%>  
+	
```

Comment:
Good use of common method. We should make one universal partial as __error.html.erb_ and use it whole application . Please do needful here.